src/lm_buddy/jobs/asset_loader.py

```diff
@@ -27,8 +27,10 @@
 class HuggingFaceAssetLoader:
```

Comment:
After I removed the "artifact loader" interface a few PRs ago, these classes no longer carry state (per the deleted comment below). It might be more fitting to simply turn these into functions instead of classes.

The reason I could see a class being useful is if the asset loader had some external dependency (e.g. an S3 client from boto3) that is required to do its job. But right now, thats not the case.